src/plugins/WebpackPlugin.ts

```diff
@@ -253,7 +253,7 @@ const createConfig = (builder: Builder, spin: Spin) => {
           index.push('webpack/hot/signal.js');
         }
       }
-      index.push(builder.entry || './src/server/index.js');
+      index.push(builder.entry || './src/index.js');
```

Comment:
This is a breaking change, it cannot be done without increasing major spinjs version. Is it worth it?

Reply:
I made this change long time ago so it is not relevant anymore. Going to revert this modification.